datashuttle/utils/ds_logger.py

```diff
@@ -0,0 +1,114 @@
+import logging
+from pathlib import Path
+from typing import Any, List, Optional
+
+import fancylog as package
+from fancylog import fancylog
+from rich import print as rich_print
+from rich.console import Console
+from rich.filesize import decimal
+from rich.markup import escape
+from rich.text import Text
+from rich.tree import Tree
+
+from . import utils
+
+
+def start(
+    path_to_log: Path, name: str, variables: Optional[List[Any]]
+) -> None:
+    """"""
+    fancylog.start_logging(
+        path_to_log,
+        package,
+        filename=name,
+        variables=variables,
+        verbose=False,
+        timestamp=True,
+        file_log_level="INFO",
+        write_git=False,
+        log_to_console=False,
```

Comment:
Do you not want any info sent to the console, or are you printing as well as logging?

Reply:
In the end I think I will continue with the current implementation to split the logging into the file and handle the print statements separately. The reason for this is to have a bit more flexibility in future when we might want to display messages to users in a different way (e.g. a pop-up box in a TUI).